api/src/dto/_baseDto.ts

```diff
@@ -17,7 +17,7 @@ export class _baseDto {
     @IsEnum(DocType)
     type: DocType;
 
-    @IsDate()
+    // TODO check if it's a valid timestamp
```

Comment:
When I posted it with a timestamp I got an error "needs to be instance of Date"

Reply:
Might need to add @IsNumber to prevent sending strings etc?